### PR TITLE
Enable custom page templates feature

### LINKED ISSUE #7065: Remove custom page template feature flag
## Feature Description

Remove the feature flag and all relevant code and make sure the feature has full test coverage (both automated and QA).

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## Context
Enables the custom page templates feature

## Summary
See [Figma](https://www.figma.com/file/1YM3jA9h4QGc4xLdyaXIJS/Stories-Sprint-1.7?node-id=7524%3A9254)

## Relevant Technical Choices
Currently, only the first 100 page templates are fetched by default. We should define what this number should be and if we can[ fetch all the templates](https://github.com/google/web-stories-wp/pull/7268#discussion_r618416479) or should implement pagination for the virtual list.

## To-do

- [x] Enable feature
- [x] Add Karma tests (still missing some parts to be able to add them all)

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Open the Page templates panel in the library
2. Verify the dropdown for choosing between the Default and Saved templates is present
3. Click on the "Save current page as template" button.
4. Verify that "Saved templates" section will be automatically opened and the new template is temporarily shown blue highlight around it.
5. Verify the "Save current page as template" button looks like [in Figma](https://www.figma.com/file/1YM3jA9h4QGc4xLdyaXIJS/Stories-Sprint-1.7?node-id=7524%3A9254).
6. Verify again generally that all the main actions work as expected, including removing a saved template and applying a template (saved or default). Applying a template should add it as a new Page without confirmation, removing a template should ask for confirmation.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.
## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7065

### CODE REVIEW

File: assets/src/design-system/components/snackbar/snackbarMessage.js [hunk 1]
```diff
@@ -167,7 +168,10 @@ const SnackbarMessage = ({
             <ActionButton onClick={handleAction}>{actionLabel}</ActionButton>
           )}
           {showCloseButton && (
-            <CloseButton onClick={onDismiss}>
+            <CloseButton
+              onClick={onDismiss}
+              aria-label={__('Close message', 'web-stories')}
```
Comment (barklund) [inline assets/src/design-system/components/snackbar/snackbarMessage.js]: I think just "close" is acceptable as the label here - it's inside a `role="dialog"`, so it should be clear.

**Review by barklund — COMMENTED** [review]
Excellent! 🤝

**Review by barklund — APPROVED** [review]
(sry, forgot to approve 🤷‍♂️ )

File: includes/Experiments.php [hunk 2]
```diff
@@ -387,17 +387,6 @@ public function get_experiments() {
...
-			 * Author: @miina
-			 * Issue: #7055
-			 * Creation date: 2021-04-09
-			 */
-			[
-				'name'        => 'customPageTemplates',
-				'label'       => __( 'Custom Page Templates', 'web-stories' ),
-				'description' => __( 'Enable adding custom page templates', 'web-stories' ),
-				'group'       => 'editor',
-			],
```
Comment (swissspidy) [inline includes/Experiments.php:400]: Nit: I usually advocate for first setting `default => true` in a PR like this, and then do the complete removal of `useFeatures` etc. in a separate PR.

This makes reverting easier should there be any significant errors.

Comment (miina) [reply includes/Experiments.php:400]: Hm, alright, will add it back just in case.

**Pull request merged**